ceph.spec.in

```diff
@@ -50,11 +50,11 @@
 %endif
 %endif
 %bcond_with seastar
-%if 0%{?fedora} || 0%{?suse_version} >= 1500
-# distros where py2 is _optional_
+%if 0%{?fedora} >= 29 || 0%{?suse_version} >= 1500
```

Comment:
@smithfarm could you amend the commit message with the one at https://github.com/ceph/ceph/commit/6639ffe3a18ebda282515580544e636e45dd20cd ?

i also ran into the OpenSSL issue when running cmake on fc29.

Reply:
Done - thanks.